shared/device.h

```diff
@@ -6,12 +6,16 @@
     #define DEVICE_ID_UHK60V1 1
     #define DEVICE_ID_UHK60V2 2
 
+    #define DEVICE_COUNT 2
+
 #ifndef DEVICE_ID
     #error "DEVICE_ID is undefined!"
 #endif
 
+#if (1-DEVICE_ID-1+DEVICE_ID-0 == 0) //ensures that DEVICE_ID is "defined" with a specific value
```

Comment:
This doesn't read well. What issue do you address here exactly?

Reply:
It seems that in modules, DEVICE_ID is defined just as `#define DEVICE_ID`, which makes that line crash when including the file. That line figures that out, as in that case, result is 2 rather than 0, since `--1 = +1`.

I included `device.h` it in `versions.h` in order to know `#define DEVICE_COUNT 2`. 

`device.h` is placed in `shared`, so it looks like it is meant to be included even in module code if needed.

Happy to hear your suggestion on what to do with it!